script.js

```diff
@@ -1,12 +1,28 @@
 function menuopen() {
```

Comment:
- [ ] Kindly use the `ES6` syntax for your function `menu-open`. The **function** keyword is indeed a working keyword but it is not javascript standardised now. Following the latest `ES-next` javascript standard will help you to be up to date and easily find out the solution for a problem if you are stuck at some place. Please I would recommend you to use the best practices in your work.